portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACNavigation.path

```diff
@@ -27,6 +27,21 @@
 	<td></td>
 </tr>
 <!-- TABS/OPTIONS AVAILABLE IN DIFFERENT PLACES E.G: ASSETS, SEGMENTS, SITES -->
+<tr>
+	<td>DEFINITIONS_ATRIBUTES_TABS</td>
+	<td>//ul[contains(@class,'nav-root')]//b[text()='${key_tabName}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ENTRIES_LIST_DROPDOWN_OPTION</td>
+	<td>//div[contains(@class, 'dropdown-menu') and contains (@class,'show')]//a[text()='${key_dropdownOption}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>GENERIC_AC_BUTTON</td>
+	<td>//button[contains(@class,'btn-${buttonType}')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @91nilo, I think you could put it in the `ACUtils` path like you did with the macro. What do you think?